<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgConcepts</class>
 <widget class="QWidget" name="wdgConcepts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/books.png</normaloff>:/xulpymoney/books.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QLabel" name="lblTotal">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Concepts report</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="wdgYearMonth" name="wdgYM" native="true"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tabExpenses">
        <attribute name="title">
         <string>Expenses</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="layExpenses">
           <item>
            <widget class="mqtw" name="mqtwExpenses" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabIncomes">
        <attribute name="title">
         <string>Incomes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="layIncomes">
           <item>
            <widget class="mqtw" name="mqtwIncomes" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionHistoricalReport">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/history.png</normaloff>:/xulpymoney/history.png</iconset>
   </property>
   <property name="text">
    <string>Historical report</string>
   </property>
   <property name="toolTip">
    <string>Historical report</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgYearMonth</class>
   <extends>QWidget</extends>
   <header location="global">wdgYearMonth</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mqtw</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
